cmd/protoc-gen-go-tetragon/eventchecker/field.go

```diff
@@ -209,6 +209,10 @@ func doGetFieldFrom(field *Field, g *protogen.GeneratedFile, handleList, handleO
 		return doListFrom()
 	}
 
+	if handleList && field.Desc.IsMap() {
+		return "// TODO: implement fromMap", nil
```

Comment:
Is it enough the implement these checks later?

Reply:
@sharlns It's on my TODO list, but I don't think it has to block this PR. FWIW I think this is still WIP.